thinkful-ei-dragonfly/stay-informed-client
# checked package dependencies for --production

-removed fortawesome/fontawesome dependencies,
-moved unfetch to production dependency 

To test =>
-delete entire /node_module directory AND package-lock.json in root (DONT DELETE PACAKGE.JSON) 
-run 'npm install --production'
-open app and make sure everything still works
